spring-cloud-gcp-vision/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
What does this do?

Reply:
Allows you to mock final classes or final methods: https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#mock-the-unmockable-opt-in-mocking-of-final-classesmethods

This is used to mock the ImageAnnotatorClient to unit test the exception handling.